yarn.lock

```diff
@@ -2,10 +2,10 @@
 # yarn lockfile v1
 
 
-"@types/lodash@4.14.194":
-  version "4.14.194"
-  resolved "https://registry.yarnpkg.com/@types/lodash/-/lodash-4.14.194.tgz#b71eb6f7a0ff11bff59fc987134a093029258a76"
-  integrity sha512-r22s9tAS7imvBt2lyHC9B8AGwWnXaYb1tY09oyLkXDs4vArpYJzw09nj8MLx5VfciBPGIb+ZwG0ssYnEPJxn/g==
+"@types/lodash@4.14.193":
+  version "4.14.193"
+  resolved "https://registry.yarnpkg.com/@types/lodash/-/lodash-4.14.193.tgz#0ca0c10002983d25f6bc86c98fd703b21ba57b58"
+  integrity sha512-DWZ7ccyFsIYfwOlYfR6amRP9PfPmvqo0UluncNbu53W62GFwWT6rzgO3HzKaNbahUZTu/66OaQU3/JrtDo4X9w==
 
 lodash@4.17.20:
```

Comment:
Notably, git auto-merged the new change that was present on master during the merge; namely, the change in `lodash`'s version from `4.17.21` to `4.17.20`. From `0892d85` on the branch `change-types-lodash-version` (its branch point from `main`), I merged `0dbd681` (the later updated `main`).

```
*   eea5f3c (change-types-lodash-version) Commit dirty merge to demonstrate what git can automatically merge
|\  
| * 0dbd681 (main) Change lodash version
* | 0892d85 Change @types/lodash version
|/  
* ba1a8e6 Initial commit
```